### ISSUE TITLE
HTML docs: placeholder strings should be italicized or underlined in HTML

### ISSUE THREAD

**Issue by floam** [fish-shell/fish-shell#8670]
I think it would be ideal if e.g. STRING, SEP, and brethren were styled differently than options here:

<img width="821" alt="Screen Shot 2022-01-28 at 3 22 22 AM" src="https://user-images.githubusercontent.com/291142/151538641-2082dc27-18ab-47af-9d68-77398916c6f2.png">

It does happen in the manpages:

<img width="514" alt="image" src="https://user-images.githubusercontent.com/291142/151538706-e234c417-3a73-4a17-a32a-993319bb218d.png">

**Comment by floam** [comment]
(Also, it would be great if we could get at least the first one [ … ] portion to be some kind of inline-block that magically aligned horizontally with all the others. Or literally a table.)

**Comment by floam** [comment]
My brain kinda likes this, but it's a lot of lines:

<img width="476" alt="Screen Shot 2022-01-28 at 3 56 16 AM" src="https://user-images.githubusercontent.com/291142/151543205-82301089-1560-4278-97ab-891d984d08f7.png">

**Comment by krobelus** [comment]
> I think it would be ideal if e.g. STRING, SEP, and brethren were styled differently than options here:

Yep, especially to set them apart from literal options that happen to be uppercase (`-N`)

> (Also, it would be great if we could get at least the first [ … ] portion to be some kind of inline-block that magically aligned horizontally with all the others. Or literally do it as a table.)

`string` is a special case; maybe we should not even disclose specific options here

```
string collect [OPTIONS] [STRING ...]
string escape  [OPTIONS] [STRING ...]
string replace [OPTIONS] PATTERN REPLACEMENT [STRING ...]
```

while `man string collect` still has the full synopsis.

**Comment by floam** [comment]
Yeah, most other utilities would probably do _OPTIONS_ as you suggest and not enumerate the entire range of switches (or at least only disclose the short ones) for the synopsis. We're unhelpfully verbose here.
